platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -251,9 +249,13 @@ sep10:
   ## @param: web_auth_domain
   ## @type: string
   ## The `web_auth_domain` property of SEP-10. https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#response
-  ## If the `web_auth_domain` is not specified, the `web_auth_domain` will be set to the domain of the value of the `host_url`.
+  ## If the `web_auth_domain` is not specified, the `web_auth_domain` will be set to the domain of the value of the `home_domain`.
   #
   web_auth_domain:
+  ## @param: home_domain
+  ## @type: string
+  ## The `home_domain` property of SEP-10. https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#request
+  home_domain: localhost:8080
```

Comment:
Can we clarify that this should be the host of the toml file? The business can technically host their toml file somewhere else. And if the toml file is not hosted by the platform, the business needs to configure `web_auth_domain` to be the host of the SEP server.

Reply:
Good idea. We will need to update stellar-docs too with this information